src/components/picker/picker.tsx

```diff
@@ -106,6 +106,12 @@ export class Picker {
     @Prop()
     public multiple: boolean = false;
 
+    /**
+     * Sets delimiters between chips.
+     */
+    @Prop()
```

Comment:
```suggestion
    @Prop({ reflect: true })
```

`fixup! feat(picker): make delimiter of chip-set available in picker`